src/main/resources/site/mixins/form-intermediate-step-data/form-intermediate-step-data.xml

```diff
@@ -3,99 +3,192 @@
         <field-set>
             <label>Mellomsteg</label>
             <items>
-                <input name="editorial" type="HtmlArea">
-                    <label>Redaksjonelt innhold ovenfor valgene</label>
-                    <occurrences minimum="0" maximum="1"/>
-                </input>
-                <input name="stepsHeadline" type="TextLine">
-                    <label>Overskrift til stegene</label>
-                    <occurrences minimum="0" maximum="1"/>
-                </input>
+                <mixin name="html-area" />
                 <item-set name="steps">
                     <label>Enkeltsteg</label>
-                    <occurrences minimum="1" maximum="0"/>
+                    <occurrences minimum="1" maximum="0" />
                     <items>
-                        <mixin name="form-intermediate-step-data-action" />
+                        <mixin name="form-intermediate-step-data-meta" />
                         <option-set name="nextStep">
                             <label>Dette valget sender brukeren videre til:</label>
                             <occurrences minimum="1" maximum="1" />
                             <options minimum="1" maximum="1">
                                 <option name="internal">
-                                    <label>Intern lenke til skjema, søknad eller klage</label>
-                                    <help-text>Brukeren skal sendes til dokument som ligger på Enonic XP</help-text>
+                                    <label><![CDATA[Intern lenke til skjema, søknad eller klage]]></label>
```

Comment:
<![CDATA[ brukes for å hindre at ting blir delt opp i flere linjer. Teksten forsvant helt i enonic studio på de nøstede stegene.